modules/installation-azure-regions.adoc

```diff
@@ -5,6 +5,32 @@
 [id="installation-azure-regions_{context}"]
 = Supported Azure regions
 
-The list of available Microsoft Azure regions is dynamically generated.
+The installation program dynamically generates the list of available Microsoft Azure regions based on your subscription. The following Azure regions were tested in {product-title} version 4.2:
 
-//IDK if there are limits. I have a question out about this.
+* centralus (Central US)
+* eastus (East US)
+* eastus2 (East US 2)
+* northcentralus (North Central US)
+* southcentralus (South Central US)
+* westcentralus (West Central US)
+* westus (West US)
+* westus2 (West US 2)
+* uksouth (UK South)
+* ukwest (UK West)
+* francecentral (France Central)
+* francesouth (France South)
+* northeurope (North Europe)
+* westeurope (West Europe)
+* japaneast (Japan East)
+* japanwest (Japan West)
+* koreacentral (Korea Central)
+* koreasouth (Korea South)
+* eastasia (East Asia)
+* southeastasia (Southeast Asia)
+* southindia (South India)
+* centralindia (Central India)
+* westindia (West India)
+* southafricanorth (South Africa North)
+* southafricawest (South Africa West)
```

Comment:
I did not have the any quota for southafricawest (South Africa West), so we should remove it until it can be re-tested with proper quota.